apps/designer/env-check.js

```diff
@@ -14,6 +14,13 @@
 require("dotenv").config();
 
 const REQUIRED_ENVS = ["DATABASE_URL", "AUTH_SECRET"];
+const S3_KEYS = [
```

Comment:
hmm, maybe I am missing something but why is this one different than S3_REQUIRED_ENV_KEYS?

Reply:
You can't import from that module because its an es module

Reply:
oh I see, put them in a js file or json file?

Reply:
I actually see that the fact this is a js and not compiled is going to be a problem in the future, we probably need to have a ts file that compiles to a js file to be usable where it is currently used